slurm.sh

```diff
@@ -13,13 +13,6 @@ slurm_submit(){
     local condaEnv="${10}"
     local quiet="${11:-'no'}"
 
-    # Need at least the command string
-
-    if [ -z "$commandString" ]; then
-        die "Need at least a command string for SLURM submission"
-    fi
```

Comment:
Why is above being removed from the slurm functionality?

Reply:
Its moved to `atlas-slurm` as in this place it will not exit the script but move to next function `slurm_monitor_job` and hangs in there..